awx/ui_next/src/screens/Template/JobTemplateEdit/JobTemplateEdit.jsx

```diff
@@ -1,9 +1,12 @@
+/* eslint react/no-unused-state: 0 */
```

Comment:
This is meant to be temporary and will be removed in a follow-up PR when adding the component.